Src/Base/AMReX_CoordSys.cpp

```diff
@@ -164,7 +164,7 @@ CoordSys::UpperIndex(const Real* point) const noexcept
     IntVect ix;
     for (int k = 0; k < AMREX_SPACEDIM; k++)
     {
-        ix[k] = (int) ((point[k]-offset[k])/dx[k]);
```

Comment:
Maybe we should do +1 to the int, not the real number before the cast.

Reply:
I moved the +1 outside int()